src/bindings/js/node/src/tensor.cpp

```diff
@@ -138,3 +139,17 @@ Napi::Value TensorWrap::get_shape(const Napi::CallbackInfo& info) {
 Napi::Value TensorWrap::get_element_type(const Napi::CallbackInfo& info) {
     return cpp_to_js<ov::element::Type_t, Napi::String>(info, _tensor.get_element_type());
 }
+
+
+
+Napi::Value TensorWrap::get_size(const Napi::CallbackInfo& info) {
+    if (info.Length() > 0) {
+        reportError(info.Env(), "getSize() does not accept any arguments.");
+        return info.Env().Null();
+    }
+    size_t size = _tensor.get_size();
+
+    double jsSize = static_cast<double>(size);
```

Comment:
Use snake_case in c++ part

Reply:
@almilosz Thanks , for the feedback I have made the suggested changes.